src/Package/Impl/DataInspect/VariableView.xaml.cs

```diff
@@ -219,5 +273,85 @@ public partial class VariableView : UserControl, IDisposable {
                     return KnownMonikers.BulletList;
             }
         }
+        #endregion
+
+        #region ICommandTarget
+        public CommandStatus Status(Guid group, int id) {
```

Comment:
Or expose VisualGrid via some sort of interface. It may be better to do it when we turn grid to a visual component.